<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainControlClass</class>
 <widget class="QMainWindow" name="MainControlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>844</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1277</width>
    <height>844</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>仿制网易云</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../res/标题栏/标题.png</normaloff>../../res/标题栏/标题.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>844</height>
    </size>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>249</width>
      <height>672</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>249</width>
      <height>672</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>249</width>
       <height>672</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_find_music_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>发现音乐</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_podcast_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>播客</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_video_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>视频</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_follow_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>关注</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_seeding_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>直播</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_roaming_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>私人漫游</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>43</height>
         </size>
        </property>
        <property name="text">
         <string>我的音乐</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="btn_like_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>我喜欢的音乐</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/zh'n/res/左侧导航栏/我喜欢.png</normaloff>C:/Users/zh'n/res/左侧导航栏/我喜欢.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="btn_local_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>本地与下载</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/zh'n/res/左侧导航栏/本地与下载.png</normaloff>C:/Users/zh'n/res/左侧导航栏/本地与下载.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="btn_recently_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>最近播放</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/zh'n/res/左侧导航栏/最近播放.png</normaloff>C:/Users/zh'n/res/左侧导航栏/最近播放.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="btn_cloud_disk_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>我的音乐云盘</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/zh'n/res/左侧导航栏/云盘.png</normaloff>C:/Users/zh'n/res/左侧导航栏/云盘.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="btn_my_podcast_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>我的播客</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/zh'n/res/左侧导航栏/我的播客.png</normaloff>C:/Users/zh'n/res/左侧导航栏/我的播客.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="btn_collection_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>我的收藏</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/zh'n/res/左侧导航栏/收藏.png</normaloff>C:/Users/zh'n/res/左侧导航栏/收藏.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>1028</width>
      <height>672</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1028</width>
      <height>672</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tab_widget_find_music_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabBar::tab { 
	font: 12pt &quot;宋体&quot;;
	border: none;
	margin-right: 20px;
	margin-left: 40px;
	min-width: 100px;
	min-height: 40px;
	padding-bottom: -1px;
}
QTabBar::tab:selected { 
	border-bottom: 5px solid red;
	font-weight: bold;
	font: 14pt &quot;黑体&quot;;
}
QTabWidget::pane {
	border: none;
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>个性推荐</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>专属订制</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>歌单</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>排行榜</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>歌手</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>最新音乐</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="playMusicWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>669</y>
      <width>1280</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>96</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainControl.qrc"/>
 </resources>
 <connections/>
</ui>
